tests/async/test_tracing.py

```diff
@@ -103,3 +103,18 @@ async def test_should_support_a_buffer_without_a_path(
     await page.goto(server.PREFIX + "/grid.html")
     trace = await browser.stop_tracing()
     assert "screenshot" in trace.decode()
+
+
+@pytest.mark.only_browser("chromium")
```

Comment:
this is different from the existing tracing, so it's not Chromium specific. Also it might make sense to move the existing tracing into a `test_chromium_tracing.py` and the new tracing into `test_tracing.py`

Reply:
Please drop the marker, as Max says.